### PR TITLE
[Debugger] Add null check for session

### PR DESCRIPTION
**Pull request by ketttY**
Fixes: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/937030

### CODE REVIEW

**Comment by sgmunn** [comment]
re-triggering build to kick off failed tests due to release definition issue.

**Comment by sgmunn** [comment]
Microsoft.VSForMac.Tests.SimpleNuGetTests.TestAddAndUpdatePackages test failure is unrelated.

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ThreadsPad.cs [hunk 1]
```diff
@@ -159,18 +159,21 @@ void ShowPopup (Gdk.EventButton evt)
...
-				pauseExecution.Clicked += delegate {
-					session.Stop ();
-				};
-				context_menu.Items.Add (pauseExecution);
+
+				if (session != null) {
+					var continueExecution = new ContextMenuItem (GettextCatalog.GetString ("Resume"));
+					continueExecution.Sensitive = !session.IsRunning;
+					continueExecution.Clicked += delegate {
+						session.Continue ();
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ThreadsPad.cs]: Should we also null check here? It's capturing `this`, so we're referring to a field that can be set to null.

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ThreadsPad.cs [hunk 2]
```diff
@@ -159,18 +159,21 @@ void ShowPopup (Gdk.EventButton evt)
...
+					};
+					context_menu.Items.Add (continueExecution);
+					var pauseExecution = new ContextMenuItem (GettextCatalog.GetString ("Pause"));
+					pauseExecution.Sensitive = session.IsRunning;
+					pauseExecution.Clicked += delegate {
+						session.Stop ();
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/ThreadsPad.cs]: And here

**Pull request merged**